Test/VimCoreTest/CommonOperationsIntegrationTest.cs

```diff
@@ -44,14 +44,14 @@ protected ScrollOffsetTest()
 
             private void AssertFirstLine(int lineNumber)
             {
-                TestableSynchronizationContext.RunAll();
+                Dispatcher.DoEvents();

```

Comment:
These all had to change, and they might have to change again. Conceptually, we are performing some kind of pumping applicable to the test infrastructure. Exposing a single class method to do that (which would currently call `Dispatcher.DoEvents()` seems like it might prevent the need for any future changes to tests doing this kind of testing.

Reply:
That's a good point. Let me try that.